spec/system/placements/schools/partner_providers/remove_a_provider_spec.rb

```diff
@@ -0,0 +1,208 @@
+require "rails_helper"
+
+RSpec.describe "User removes a provider from providers list",
+               service: :placements, type: :system do
+  include ActiveJob::TestHelper
+
+  around do |example|
+    perform_enqueued_jobs { example.run }
+  end
+
+  scenario do
+    given_providers_exist
+    and_i_am_signed_in
+
+    when_i_am_on_the_providers_index_page
+    and_i_click_on_springfield_university_in_the_providers_list
+    then_i_see_the_show_page_for_springfield_university
+
+    # User clicks cancel link
+    when_i_click_on_delete_provider
+    and_i_am_asked_to_confirm_delete_springfield_university
+    and_i_click_on_cancel
+    then_i_see_the_provider_index_page
```

Comment:
Could we split this block into two?

```suggestion
    when_i_click_on_delete_provider
    then_i_am_asked_to_confirm_delete_springfield_university
    
    when_i_click_on_cancel
    then_i_see_the_provider_index_page
```